pkg/controller/infinispan/resources/secret/secret_controller.go

```diff
@@ -81,17 +81,13 @@ func Add(mgr manager.Manager) error {
 
 func (s *secretResource) Process() (reconcile.Result, error) {
 	// Reconcile Encryption Secrets
-	if result, err := s.reconcileKeystoreSecret(); result != nil {
-		return *result, err
-	}
-
```

Comment:
it's not clear to me why we don't need this any more

Reply:
This function just set owner reference in the code, nothing more

Reply:
oh keystore is never generated, right?

Reply:
Seems to yes